charts/network-agent/Chart.yaml

```diff
@@ -7,3 +7,12 @@ home: https://honeycomb.io
 sources:
   - https://github.com/honeycombio/honeycomb-network-agent
 icon: https://www.honeycomb.io/wp-content/themes/honeycomb/assets/img/logo.svg
+maintainers:
```

Comment:
Should we use an alias email instead? eg collections@honeycomb.io or for the network agent, telemetry@honeycomb.io as we're still maintaining the project?

Reply:
Updated